net.zdechov.app.C-evo.yml

```diff
@@ -0,0 +1,50 @@
+app-id: net.zdechov.app.C-evo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: c-evo
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - name: c-evo
+    sources:
+      - type: svn
+        url: https://svn.zdechov.net/c-evo/tags/1.3.6
+        revision: 634
+    buildsystem: simple
+    build-commands:
+      - |
+        . /usr/lib/sdk/freepascal/enable.sh
+        lazbuild --build-mode=Release AI/StdAI/StdAI.lpi
+        mv AI/StdAI/libstdai.so AI/StdAI/libstdai-amd64.so
```

Comment:
what happen on aarch64?

Reply:
Good point. I have fixed the naming of StdAI library based on the architecture. Compilation itself should be ok.

Reply:
why does it need a different naming?

Reply:
That is how it is currently done. Each AI has its own definition file where multiple architectures can be specified so the author of the particular AI can distribute his AI as closed source with multiple pre-compiled libraries for different architectures. StdAI as standard open source implementation has them pre-compiled in the repository so main project can be built and run without need to always recompile it for given architecture. If the app is packaged for particular architecture then it doesn't matter as much but still compiled library needs to have matching name so the game knows what to load.

#PATH StdAI.dll
#PATH_WIN32 StdAI-win32.dll
#PATH_WIN64 StdAI-win64.dll
#PATH_LINUX_I386 libstdai-i386.so
#PATH_LINUX_AMD64 libstdai-amd64.so
#PATH_LINUX_ARM32 libstdai-arm32.so
#PATH_LINUX_ARM64 libstdai-arm64.so